app.js

```diff
@@ -12,6 +13,9 @@ app.use("/api", express.json());
 // Enable cors
 app.use("/api", cors());
 
+// enable logging
+app.use(morgan('combined'))
```

Comment:
I just read through the doc page, definitely looks like it will do the job. I remember you said logging would be in loki or grafana or something, but I don't remember where that would be 😵‍💫  Can't wait to see it in action